toolchain/check/cpp/thunk.cpp

```diff
@@ -415,8 +438,8 @@ static auto BuildCalleeArgs(clang::Sema& sema,
   // The object parameter is always passed as `self`, not in the callee argument
   // list, so the first argument corresponds to the second parameter if there is
   // an explicit object parameter and the first parameter otherwise.
-  unsigned first_param = callee_info.has_explicit_object_parameter();
-  unsigned num_params = callee_info.decl->getNumParams();
+  int first_param = callee_info.has_explicit_object_parameter();
+  int num_params = callee_info.num_params;
```

Comment:
You might just remove this local, I assume it was added because the previous call was longer (and a function call)